src/components/LoanCards/LoanTags/LoanTagV2.vue

```diff
@@ -0,0 +1,66 @@
+<template>
+	<div v-if="!!variation && timeLeftMs > 0" class="tw-text-small tw-font-medium tw-mt-0.5" style="color: #CE4A00;">
```

Comment:
Add class `tw-line-clamp-1`, since the ticket mentions truncating to 1 line.

Reply:
Also it looks like the mock currently have the color `#FF5C00`

Reply:
![Screen Shot 2023-02-01 at 3 48 48 PM](https://user-images.githubusercontent.com/15318176/216159980-4aaf6a1a-b8b7-4500-96af-91c0158b5812.png)
🤔  looks like there's two diff ones 😂  I think it's the one I used here since that one is used more throughout the figma but i'll double check w/ Nathan

Reply:
Interesting, I kept seeing the other color. Let me know when you find out, since the other work with orange will also need to be adjusted.